app/models/guide.rb

```diff
@@ -2,7 +2,19 @@ class Guide
   include Mongoid::Document
   include Mongoid::Paperclip
   include Mongoid::Slug
-  searchkick
+  searchkick callbacks: :async, merge_mappings: true, mappings: {
+    guide: {
+      properties: {
+        compatibilities: {
+          type: 'nested',
+          properties: {
+            user_id: {type: "string"},
+            score: {type: "integer"}
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.